code/controllers/subsystem/persistence/_persistence.dm

```diff
@@ -349,3 +350,25 @@ SUBSYSTEM_DEF(persistence)
 		if(!ending_human.client)
 			return
 		ending_human.client.prefs.save_character()
+
+/datum/controller/subsystem/persistence/proc/SaveTCGCards()
+	for(var/i in GLOB.joined_player_list)
+		var/mob/living/carbon/human/ending_human = get_mob_by_ckey(i)
+		if(!istype(ending_human) || !ending_human.mind || !ending_human.client || !ending_human.client.prefs || !ending_human.client.prefs.tcg_cards)
+			continue
+
+		var/mob/living/carbon/human/original_human = ending_human.mind.original_character
+		if(!original_human || original_human.stat == DEAD || !(original_human == ending_human))
+			continue
+
+		var/obj/item/tcgcard_binder/binder = locate() in ending_human
+		if(!binder || !length(binder.cards))
+			continue
+
+		var/list/card_types = list()
+		for(var/obj/item/tcg_card/card in binder.cards)
+			if(!card.illegal) //Nope, don't want free cards as traitor/ops!
+				card_types.Add(card.datum_type)
+
+		ending_human.client.prefs.tcg_cards = card_types
+		ending_human.client.prefs.save_character()
```

Comment:
It'd be nice if you found a way to do this without having to call save_character, as this also saves all the stuff you changed in the round, but *didn't* want to permanently save (e.g, changing your job prefs for one round, etc etc)

Reply:
Yeah I ca do that.

Reply:
Theres a problem. I _can_ do that but it would be really lots of shitcode and new save files

Reply:
Maybe @silicons knows a good way to do that, since it's pretty annoying from experience, given that persistant scars also do it if enabled.